log/development.log

```diff
@@ -1,12 +1,7 @@
   [1m[35m (0.1ms)[0m  [1m[34mSELECT pg_try_advisory_lock(1580829076522303230)[0m
-  [1m[35m (0.8ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
-  [1m[36mActiveRecord::InternalMetadata Load (0.6ms)[0m  [1m[34mSELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", "environment"], ["LIMIT", 1]]
+  [1m[35m (0.3ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
```

Comment:
remove this file. 🙏 
![](https://media.giphy.com/media/cNatnV637tPW0/giphy-downsized.gif)